implementation.js

```diff
@@ -138,6 +152,9 @@ module.exports = function from(arrayLike) {
 	}
 	var items = ES.ToObject(arrayLike);
 
+	// variables for rebuilding array from iterator or string.
+	items = parseES6(items);
```

Comment:
Rather than reassign, can line 153 use a different var name, so this line can be `var items`?